src/components/AppProvider/AppProvider.tsx

```diff
@@ -100,10 +100,12 @@ export class AppProvider extends React.Component<AppProviderProps, State> {
 
   render() {
     const {theme = {}, features = {}, children} = this.props;
+    const {newDesignLanguage = false, ...restOfFeatures} = features;
```

Comment:
I think we can dodge having to destructure to create some local variables then recombine them into into an object. I believe this has the same effect

```
const {theme = {}, children} = this.props;
const features = {newDesignLanguage: false, ...this.props.features};

// Later
 <FeaturesContext.Provider value={features}>
```

Same can be done in the test provider